roachlib/db.cc

```diff
@@ -235,7 +237,7 @@ class DBCompactionFilter : public rocksdb::CompactionFilter {
       // Transaction rows are GC'd if their timestamp is older than
       // the system-wide minimum write intent timestamp. This
       // system-wide minimum write intent is periodically computed via
-      // map-reduce over all ranges and gossipped.
```

Comment:
The changes in this file were mostly to correct misspelling of "gossipped" and remove trailing whitespace.